panther_hardware_interfaces/include/panther_hardware_interfaces/panther_imu_ros_interface.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PANTHER_HARDWARE_INTERFACES_PANTHER_IMU_ROS_INTERFACE_HPP_
+#define PANTHER_HARDWARE_INTERFACES_PANTHER_IMU_ROS_INTERFACE_HPP_
+
+#include <atomic>
+#include <functional>
+#include <memory>
+#include <thread>
+
+#include <rclcpp/rclcpp.hpp>
+#include <std_srvs/srv/trigger.hpp>
+#include <std_msgs/msg/bool.hpp>
+
+#include <phidgets_spatial_parameters.hpp>
+
+namespace panther_hardware_interfaces
+{
+
+using TriggerSrv = std_srvs::srv::Trigger;
+
+/**
+ * @brief Class that takes care of additional ROS interface of panther imu, such as providing service for calibration
+ */
+class PantherImuRosInterface
+{
+public:
+  /**
+   * @brief Creates node and executor (in a separate thread), publishers, subscribers and services
+   *
+   * @param calibrate - functions that should be called, when calibration is triggered
+   * service is called
```

Comment:
```suggestion
    * @param calibrate - function to be called when the calibration service is triggered
```
Is it the proper meaning of this sentence?

Reply:
Yup this sounds weird